### PR TITLE
Adds the project demoqa to the repository.

### PR DESCRIPTION
**Pull request by renekounkou**

### CODE REVIEW

File: demoqa/src/main/java/framework/SeleniumTestBase.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+public abstract class SeleniumTestBase {
+	private WebDriver driver;
+	
+	@BeforeTest
+	public void setup() {
+	System.setProperty("webdriver.chrome.driver", "C:\\ChromeDriver\\chromedriver.exe");
+	
+	this.driver =  new ChromeDriver();
+	
+	setOptions();
```
Comment (dentaylor) [inline demoqa/src/main/java/framework/SeleniumTestBase.java:19]: Remove dead code (this method does nothing)

File: demoqa/src/main/java/framework/SeleniumTestBase.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		
+	}
+		protected WebDriver getDriver() {
+			return this.driver;
+	}
+
+		private void setOptions() {
+	    this.driver.manage().window().maximize();
+	    this.driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
+
```
Comment (dentaylor) [inline demoqa/src/main/java/framework/SeleniumTestBase.java:36]: Fix indentation throughout your code base

File: demoqa/src/test/java/demoqa/com/ControlExtensionBase.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package demoqa.com;
+
+import org.openqa.selenium.WebElement;
+
+public abstract class ControlExtensionBase {
```
Comment (dentaylor) [inline demoqa/src/test/java/demoqa/com/ControlExtensionBase.java:5]: You should add a constructor here that initializes wrappedElement.

File: demoqa/src/test/java/demoqa/com/ControlExtensionFactory.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
+package demoqa.com;
+
+import org.openqa.selenium.WebElement;
+
+public class ControlExtensionFactory {
```
Comment (dentaylor) [inline demoqa/src/test/java/demoqa/com/ControlExtensionFactory.java:5]: This factory should only return one instance of the type of control extension.

File: demoqa/src/test/java/demoqa/com/DemoQaTests.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+import org.testng.annotations.Test;
+
+import framework.SeleniumTestBase;
+
+public class DemoQaTests extends SeleniumTestBase {
+  @Test
+  public void canSelectRadioButton() {
+	  
+	  RadioButtonPage page = new RadioButtonPage(this.getDriver());
+	  page.navigate();
```
Comment (dentaylor) [inline demoqa/src/test/java/demoqa/com/DemoQaTests.java:15]: Implement the fluent page object pattern

**Review by dentaylor — APPROVED** [review]

File: demoqa/src/main/java/framework/SeleniumTestBase.java (hunk 1 above)
Comment (renekounkou) [reply demoqa/src/main/java/framework/SeleniumTestBase.java:19]: Resolved

File: demoqa/src/main/java/framework/SeleniumTestBase.java (hunk 2 above)
Comment (renekounkou) [reply demoqa/src/main/java/framework/SeleniumTestBase.java:36]: Resolved

File: demoqa/src/test/java/demoqa/com/ControlExtensionBase.java (hunk 3 above)
Comment (renekounkou) [reply demoqa/src/test/java/demoqa/com/ControlExtensionBase.java:5]: Resolved.

**Pull request merged**